README.md

```diff
@@ -24,6 +24,10 @@ Add the following to your `mbed_app.json` file:
 }
 ```
 
+### UBLOX_EVK_ODIN_W2 / Ethernet
+
```

Comment:
- Why underscores in the name? At the official ublox site the name would be ublox EVK-ODIN-W2
- You don't need  spaces around the slash.

Reply:
That's the official board target name in `mbed cli` and all related `mbed_app.json` mangling.